### PR TITLE
Add unit test

### PR DESCRIPTION
**Pull request by dongzl**
**What this PR does**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: pkg/util/match/slice_match_test.go [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+	listB = []interface{}{2, 3, 4}
+	extraA, extraB = diffLists(listA, listB)
+	expectedExtraA := []interface{}{1}
+	expectedExtraB := []interface{}{4}
+	if !reflect.DeepEqual(extraA, expectedExtraA) || !reflect.DeepEqual(extraB, expectedExtraB) {
+		t.Errorf("Expected extraA=%v and extraB=%v, got extraA=%v, extraB=%v", expectedExtraA, expectedExtraB, extraA, extraB)
+	}
+}
+
+// 测试 expected 和 actual 都为 nil 的情况
```
Comment (baerwang) [inline pkg/util/match/slice_match_test.go:213]: 这里是不是要改成英语比较合适

**Review by baerwang — APPROVED** [review]

File: pkg/util/env/env_test.go [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+	"testing"
+)
+
+import (
+	"github.com/arana-db/arana/pkg/constants"
+)
+
+func TestIsDevelopEnvironment(t *testing.T) {
+	// Test case 1: Environment variable is set to "1"
+	os.Setenv(constants.EnvDevelopEnvironment, "1")
```
Comment (jjeffcaii) [inline pkg/util/env/env_test.go:31]: It will be better if reset the original env using defer when tests done.

File: pkg/util/file/file_test.go [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+	if result != expected {
+		t.Errorf("IsYaml(%s) = %v; want %v", path, result, expected)
+	}
+
+	// Test with a valid YML file
+	path = "example.yml"
+	expected = true
+	result = IsYaml(path)
+	if result != expected {
+		t.Errorf("IsYaml(%s) = %v; want %v", path, result, expected)
```
Comment (jjeffcaii) [inline pkg/util/file/file_test.go:38]: use asserts.True instead.

**Review by jjeffcaii — CHANGES_REQUESTED** [review]
In short word, pls using testify lib instead.

**Review by jjeffcaii — APPROVED** [review]

**Pull request merged**